app/scripts/lib/experiments/grouping-rules/token-code.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define(function (require, exports, module) {
+  'use strict';
+
+  const BaseGroupingRule = require('./base');
+  const GROUPS = ['control', 'treatment'];
+  const ROLLOUT_CLIENTS = [{
+    clientId: 'dcdb5ae7add825d2',
+    name: 'Done321',
+    rolloutRate: 1.0 // Rollout rate between 0..1
+  }, {
+    clientId: '3a1f53aabe17ba32',
+    name: 'Firefox Add-ons',
+    rolloutRate: 0.0
+  }, {
+    clientId: 'ecdb5ae7add825d4',
+    name: 'TestClient',
+    rolloutRate: 0.0
+  }];
```

Comment:
No biggie but it looks like these are always indexed by `clientId`, is an array the right data structure?